### PR TITLE
Adding contributing guide

### LINKED ISSUE #85: Create a contributing guide for documentation
Similarly to [Kubernetes docs contributing guides](https://kubernetes.io/docs/contribute/new-content/), describe documentation contributing guides for Kubewarden.

### PR DESCRIPTION
**Pull request by divya-mohan0209**
Fixes #85 
- Drawing inspiration from the [Kubernetes contribution guide](https://kubernetes.io/docs/contribute/)
- Accessible to old and new contributors alike

### CODE REVIEW

File: src/contribution-guide/contributing.md
Comment (ereslibre) [inline src/contribution-guide/contributing.md:24]: Are this links going to be filled later on?

**Review by ereslibre — COMMENTED** [review]
Thanks @divya-mohan0209, LGTM. One comment regarding two missing links.

**Review by viccuad — APPROVED** [review]
Thanks! LGTM modulo the comment from Rafa.

File: src/contribution-guide/contributing.md
Comment (divya-mohan0209) [reply src/contribution-guide/contributing.md:24]: Hello Rafael, yes this folder will contain documents for the contributor guide. This is the front page. I'm working on it section-by-section.

**Review by jvanz — APPROVED** [review]
:clap:

File: src/contribution-guide/contributing.md
Comment (jtravee) [inline src/contribution-guide/contributing.md:14]:
```suggestion
or reporting a bug/typo with the help of a GitHub issue to the
```

Comment (jtravee) [reply src/contribution-guide/contributing.md:14]: Keeping the verb tense consistent (submitting, reporting). You might even say "by reporting" as well.

File: src/contribution-guide/contributing.md
Comment (jtravee) [inline src/contribution-guide/contributing.md:17]:
```suggestion
an essential prerequisite to getting started.
```

File: src/contribution-guide/suggesting-an-improvement.md
Comment (jtravee) [inline src/contribution-guide/suggesting-an-improvement.md:14]:
```suggestion
- Click **Submit new issue**.
```

File: src/contribution-guide/suggesting-an-improvement.md
Comment (jtravee) [inline src/contribution-guide/suggesting-an-improvement.md:15]:
```suggestion
- After submitting the issue, you can either assign it to yourself or wait for a community member to pick it up. Members of the documentation team and from the community might request clarifications before they can take action on your issue, so we request that you actively check your issue or [turn on GitHub notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications).
```

File: src/contribution-guide/suggesting-an-improvement.md
Comment (jtravee) [inline src/contribution-guide/suggesting-an-improvement.md:21]:
```suggestion
- Choose an existing page in the section you think the content belongs in and click **Create an issue**.
```

Comment (jtravee) [reply src/contribution-guide/suggesting-an-improvement.md:21]: Suggesting to make the instructions parallel (so both options start with action verb with an understood "You").

File: src/contribution-guide/suggesting-an-improvement.md
Comment (jtravee) [inline src/contribution-guide/suggesting-an-improvement.md:27]:
```suggestion
No contribution is too big or small! However, to ensure that the community derives maximum value, we request that you follow the below when reporting an issue:
```

File: src/contribution-guide/suggesting-an-improvement.md
Comment (jtravee) [inline src/contribution-guide/suggesting-an-improvement.md:33]:
```suggestion
- Check for an existing pull request or issue as there is a fair chance that you're opening an issue that has been referenced before. Ensure you reference the existing issue or pull request within the issue you're opening to provide context for contributors who may want to work on it.
```

Comment (jtravee) [reply src/contribution-guide/suggesting-an-improvement.md:33]: For these, suggesting to make the sentences parallel with common verb beginnings and tenses for easier comprehension.

File: src/contribution-guide/contributing.md
Comment (jtravee) [inline src/contribution-guide/contributing.md:11]: This is super minor, but in all other headers, you have followed the style of capping the first and proper words and leaving the rest lowercase. As such, this can be **Getting started**.

**Review by jtravee — CHANGES_REQUESTED** [review]

**Review by raulcabello — APPROVED** [review]
LGTM, thanks!

**Comment by flavio** [comment]
@divya-mohan0209 we moved the card to blocked because you have been traveling around the world :)

We don't want to merge the suggestions of @jtravee behind your back.

**Comment by divya-mohan0209** [comment]
@flavio : This has now been done. Please re-review I've also requested @jtravee to do the same! After this, I shall merge it & push in separate changes on the Sidebar

File: src/contribution-guide/contributing.md
Comment (flavio) [inline src/contribution-guide/contributing.md:22]: @divya-mohan0209 this is referencing mdBook, which we don't use anymore. Maybe we can reword the sentence, after all we're still using markdown

**Review by flavio — CHANGES_REQUESTED** [review]
All good, just one minor fix

File: src/contribution-guide/contributing.md
Comment (divya-mohan0209) [reply src/contribution-guide/contributing.md:22]: Thank you, Flavio for catching this! Addressed this in the latest commit!

**Review by flavio — APPROVED** [review]
LGTM

File: src/contribution-guide/contributing.md
Comment (jtravee) [inline src/contribution-guide/contributing.md:23]: We generally use the "or" ending in the American style vs. the "our" ending in the British style.

File: src/contribution-guide/suggesting-an-improvement.md
Comment (jtravee) [inline src/contribution-guide/suggesting-an-improvement.md:5]:
```suggestion
In most cases, any new documentation work for Kubewarden begins with an issue in GitHub. The [documentation team for Kubewarden](https://github.com/orgs/kubewarden/teams/kubewarden-documentation) reviews, categorizes, and tags them as appropriate. Everybody is welcome to work on the issue, including the reporter, but we request that you assign it to yourself before commencing any work to avoid duplicate efforts.
```

File: src/contribution-guide/suggesting-an-improvement.md
Comment (jtravee) [inline src/contribution-guide/suggesting-an-improvement.md:15]:
```suggestion
- After submitting the issue, you can either assign it to yourself or wait for a community member to pick it up. Members of the documentation team and the community might request clarifications before they can take action on your issue, so we request that you actively check your issue or [turn on GitHub notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications).
```

File: docs/contribution-guide/contributing.md
Comment (jtravee) [inline docs/contribution-guide/contributing.md:23]:
```suggestion
Ensure you're familiar with Markdown before you begin.
```

Comment (jtravee) [reply docs/contribution-guide/contributing.md:23]: If you remove "flavor", then we can also remove the preceding "the" in this case.

**Review by jtravee — CHANGES_REQUESTED** [review]

**Review by jtravee — APPROVED** [review]

**Comment by divya-mohan0209** [comment]
Merging the pull request.

**Pull request merged**
